src/sync.cpp

```diff
@@ -395,15 +395,66 @@ namespace librealsense
                     // something missing, we can't release anything yet...
                     for( auto i : missing_streams )
                     {
-                        LOG_IF_ENABLE( "... missing " << i->get_name() << ", next expected "
-                                                      << _next_expected[i],
+                        LOG_IF_ENABLE( "... missing " << i->get_name() << ", next expected @" << _next_expected[i].value
+                                                      << " (from " << _next_expected[i].fps << " fps)",
                                        env );
                         if( skip_missing_stream( *curr_sync, i, last_arrived, env ) )
                         {
-                            LOG_IF_ENABLE( "...     ignoring it", env );
+                            LOG_IF_ENABLE( "...     cannot be synced; not waiting for it", env );
```

Comment:
Can we add which frame cannot be synced?

Reply:
It says it right before:
```
... have [Color/1 #3 @300.00]
... missing Depth/0, next expected @190 (from 100 fps)
...     exceeded cutout of {NE+7*gap}890; deactivating matcher!
...     cannot be synced; not waiting for it
```